data/ir_cron_data.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="purchase_order_cron" model="ir.cron">
+        <field name="name">cron.archive.purchase.order</field>
```

Comment:
A cron job itself should have a user-friendly name to show on the UI, not like a normal view. So please rename this cron job